client/app/telecom/telephony/fax/fax/convertToVoicefax/telecom-telephony-fax-fax-convertToVoicefax.js

```diff
@@ -5,7 +5,11 @@ angular.module("managerApp").config(function ($stateProvider) {
         views: {
             "@telephonyView": {
                 templateUrl: "app/telecom/telephony/fax/fax/convertToVoicefax/telecom-telephony-fax-fax-convertToVoicefax.html",
-                noTranslations: true
+                noTranslations: true,
+                controller: function ($stateParams) {
```

Comment:
I know that it is a small part of code. But, can you create the appropriate file for the controller please. Just to be compliant with the rest of the manager ;-)